### PR TITLE
Add compatibility for 3-Groups RCU from EGLO

### PR DESCRIPTION
**Pull request by fsedarkalex**
Please review and test compile, I don't know much about TS so might have syntax errors :)

### CODE REVIEW

**Comment by fsedarkalex** [comment]
@Koenkk this is the requested PR for #5853

File: src/devices/awox.ts [hunk 1]
```diff
@@ -97,6 +104,27 @@ const definitions: Definition[] = [
             'green', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up', 'brightness_move_down', 'brightness_stop',
             'recall_1', 'color_temperature_step_up', 'color_temperature_step_down'])],
     },
+    {
+        zigbeeModel: ['ERCU_3groups_Zm'],
+        fingerprint: [
```
Comment (Koenkk) [inline src/devices/awox.ts:109]: You can drop the fingerprint (identification goes via zigbeeModel)

Comment (fsedarkalex) [reply src/devices/awox.ts:109]: Did so. Hope I did it correctly :)

Comment (fsedarkalex) [reply src/devices/awox.ts:109]: But for some reason Github did not update the PR accordingly... I am a bit lost, using GitLab normally :D

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**